src/librustc_middle/ty/query/on_disk_cache.rs

```diff
@@ -909,10 +909,12 @@ where
 
 macro_rules! encoder_methods {
     ($($name:ident($ty:ty);)*) => {
-        #[inline]
-        $(fn $name(&mut self, value: $ty) -> Result<(), Self::Error> {
```

Comment:
Haha, this one is great!

Reply:
Oh dear, I just realized what's going on now.